compal/__init__.py

```diff
@@ -301,6 +110,13 @@ def post(self, path, _data, **kwargs):
 
         return res
 
+    def post_binary(self, path, binary_data, filename, **kwargs):
+        headers = {
+            'Content-Disposition':'form-data; name=\"file\"; filename=\"%s\"' % filename,
```

Comment:
This *could* be performed using requests as well, see http://www.python-requests.org/en/latest/user/quickstart/#post-a-multipart-encoded-file